client/src/app.js

```diff
@@ -51,7 +51,16 @@ export default function main({ DOM, HTTP, route, storage, scanner: scan$, search
     , last_txids: parseHashes(loc.query.txids)
     , est_chain_seen_count: +loc.query.c || 0
     }))
-  , goAssetList$ = !process.env.IS_ELEMENTS || !process.env.ASSET_MAP_URL ? O.empty() : route('/assets')
+  //, goAssetList$ = !process.env.IS_ELEMENTS || !process.env.ASSET_MAP_URL ? O.empty() : route('/assets')
+  , goAssetList$ = !process.env.ASSETLIST_URL ? O.empty() : route(process.env.ASSETLIST_URL).map(loc => {
+    return(
+        { startIndex: loc.query.start_index != null ? loc.query.start_index : 0,
```

Comment:
This and `limit` below should be cast to a number and default to 0 if it isn't a valid number, with something like `loc.query.start_index != null && +loc.query.start_index || 0`, or even just `+loc.query.start_index || 0`

Reply:
changes have been made